android/src/app/organicmaps/MwmActivity.java

```diff
@@ -1234,12 +1234,19 @@ public void adjustBottomWidgets(int offsetX)
 
     int mapButtonsHeight = 0;
     int mainMenuHeight = 0;
+    int navButtonSheetHeight = 0;
     if (mMapButtonsController != null)
       mapButtonsHeight = (int) mMapButtonsController.getBottomButtonsHeight() + mNavBarHeight;
     if (mMainMenu != null)
       mainMenuHeight = mMainMenu.getMenuHeight();
+    final View navButtonSheet = findViewById(R.id.line_frame);
+    if (navButtonSheet != null)
+      navButtonSheetHeight = navButtonSheet.getHeight();
 
+    //final int y = Math.max(Math.max(Math.max(mapButtonsHeight, mainMenuHeight), mNavBarHeight), navButtonSheetHeight);
     final int y = Math.max(Math.max(mapButtonsHeight, mainMenuHeight), mNavBarHeight);
+    Logger.w("OFFSET", "y = " + y + " mapButtonsHeight = " + mapButtonsHeight + " mainMenuHeight = " + mainMenuHeight +
+        " " + " navButtonSheetHeight = " + navButtonSheetHeight);
```

Comment:
navButtonSheetHeight is only for debug print here, or supposed to add additional logic here?
Like use "navButtonSheetHeight" in "y" calculation, no?